### PR TITLE
chore(pr): alert reviewers about breaking changes

### PR DESCRIPTION
**Pull request by lmatz**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

https://www.notion.so/Risingwave-data-directory-got-overwritten-eaa78cb886c24fee8fa536f4b5bf122c?d=a79cdca34a694c9797123dd86253c7eb#fcfd56a59c6d4b70a37fe1892e876970

## Checklist For Contributors

- [ ] I have written necessary rustdoc comments
- [ ] I have added necessary unit tests and integration tests
- [ ] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features #7934).
- [ ] I have demonstrated that backward compatibility is not broken by breaking changes and created issues to track deprecated features to be removed in the future. (Please refer to the issue)
- [ ] All checks passed in `./risedev check` (or alias, `./risedev c`)

## Checklist For Reviewers

- [ ] I have requested macro/micro-benchmarks as this PR can affect performance substantially, and the results are shown.

## Documentation

- [ ] My PR **DOES NOT** contain user-facing changes.

<details><summary>Click here for Documentation</summary>

### Types of user-facing changes

Please keep the types that apply to your changes, and remove the others.

- Installation and deployment
- Connector (sources & sinks)
- SQL commands, functions, and operators
- RisingWave cluster configuration changes
- Other (please specify in the release note below)

### Release note

</details>

### CODE REVIEW

**Comment by lmatz** [comment]
make a PR a breaking change by default, probably more likely that people will be cautious about this.

**Comment by fuyufjh** [comment]
🤔 From my understanding we should NOT have any changes that breaks backward compability. So it looks a little bit weird to me.

**Comment by lmatz** [comment]
🤣  you are right, let me rephrase it

File: .github/labeler.yml [hunk 1]
```diff
@@ -28,3 +28,7 @@ labels:
 - label: "user-facing-changes"
   negate: true
   body: 'My PR \*\*DOES NOT\*\* contain user-facing changes'
+  
+- label: "breaking-change"
+  negate: true
+  body: 'This PR \*\*IS NOT\*\* a breaking change.'
```
Comment (fuyufjh) [inline .github/labeler.yml:34]:
```suggestion
```
